Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -305,21 +308,65 @@ class OutputHelper
         m_GPUSharedStart[iterNum] = profiler[EVAL_MODEL].GetGpuSharedStart();
         m_GPUDedicatedDiff[iterNum] = profiler[EVAL_MODEL].GetGpuDedicatedDiff();
     }
+
+    void SaveResult(uint32_t iterationNum, std::string result, int hashcode)
+    {
+        m_outputResult[iterationNum] = result;
+        m_outputTensorHash[iterationNum] = hashcode;
+    }
     
-    void SetDefaultCSVFileNamePerIteration()
+    void SetDefaultFolder()
```

Comment:
Can we change this method name to capture the idea that it's for setting the folder for per iterations run data?

Maybe: "SetDefaultPerIterationFolder"